### ISSUE TITLE
ST16000VN001 EPCfeature disable not supported

### ISSUE THREAD

**Issue by Deepcuts** [Seagate/openSeaChest#41]
Hello,

I have 4 Seagate Ironwolf NAS 16 TB ST16000VN001 drives.
I would like to change the EPCfeature but for some reason Seachest reports that these drives do not support EPC

```
SeaChest_PowerControl_201_12130_64.exe -d PD2 --EPCfeature disable
EPC Feature is not supported by this device \\.\PhysicalDrive2.
```
![image](https://user-images.githubusercontent.com/4704834/96189467-315ade00-0f49-11eb-95b4-58d228af5e84.png)

Nevertheless, -i shows that EPC is enabled
![image](https://user-images.githubusercontent.com/4704834/96189492-3e77cd00-0f49-11eb-8764-b9720986fd21.png)

I have opened a ticket with Seagate but after about 5 replies I am convinced the people replying have absolutely no connection to Seagate.

**Comment by vonericsen** [comment]
Hi @Deepcuts,
Thanks for the report. This looks like a bug in detecting the EPC feature when trying to disable it. 
We may or may not already have this fixed in openSeaChest, but I will need to test and confirm it first.

If you have the ability to compile and test openSeaChest for this, that would be great, but if not, we'll look into it to make sure this is corrected before our next release.

**Comment by Deepcuts** [comment]
Hi @vonericsen ,

I was able to compile on linux.
Ran the newly compiled version and was able to disable EPC on all four drives.
No more click freakin' clack all the time after a small idle period. Ordering a Voodoo doll now for whomever came up with this EPC enabled by default on performance oriented drives.
![image](https://user-images.githubusercontent.com/4704834/96537412-de618d80-129e-11eb-8a73-575534429396.png)

Shutdown for good measure.
Back in Windows and -i shows EPC is not longer enabled.
![image](https://user-images.githubusercontent.com/4704834/96537595-5039d700-129f-11eb-881a-1ec8a88bd943.png)

Thank you for pointing me in the right direction.
